lib/rollbax/logger.ex

```diff
@@ -89,6 +89,11 @@ defmodule Rollbax.Logger do
     Logger.compare_levels(lvl, min) != :lt
   end
 
+  defp whitelisted?(whitelist, {_Logger, message, _event_time, _meta}) do
+    message = message |> prune_chardata() |> IO.chardata_to_string()
```

Comment:
This will be a duplicated message handling, which is not cheap.
I think we should check the patterns in the `post_event/3` function or build `message` content once before `post_event /3` call.